mm2src/lp_swap.rs

```diff
@@ -216,7 +218,8 @@ pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) {
                 Ok(status) => {
                     if let Err(e) = save_stats_swap(&ctx, &status.data).await {
                         error!("Error saving the swap {} status: {}", status.data.uuid(), e);
-                    }
+                    };
+                    process_coins_price_update(&ctx, &status.data).await
```

Comment:
We can upload coins prices along the swap status. As you can see, we save `SavedSwap` into the `stats_swaps` table only when the swap is finished actually. So I suggest not updating prices right after the swap status is uploaded to the database, but upload the swap status with prices.